ebml/EbmlCrc32.h

```diff
@@ -47,7 +47,7 @@ START_LIBEBML_NAMESPACE
 DECLARE_EBML_BINARY(EbmlCrc32)
   public:
     EbmlCrc32(const EbmlCrc32 & ElementToClone);
-    EbmlCrc32 &operator =(const EbmlCrc32 &) = default;
+    EbmlCrc32 &operator =(const EbmlCrc32 &) { return *this; }
```

Comment:
This doesn't seem like an improvement.

Reply:
This is a compilation fix for -std=c++98. Again, I have no idea which C++ version this project targets.

Reply:
Not only is this not an improvement, it's plain wrong! This is an assignment operator, and your change does not assign anything.

So far we haven't talked about a minimum language version. I highly suggest at least C++11; all compilers that don't support C++11 are ten years old by now. However, I'll open a new issue for a discussion about that in a moment.

Reply:
Hrm compiler suggested this to me.

I second the C++11 change. I left out all of clang-tidy’s C++11 fixes as I was unsure.